dev/cognitiveservices/data-plane/Language/questionanswering-authoring.json

```diff
@@ -285,12 +285,6 @@
           "application/json"
         ],
         "responses": {
-          "200": {
-            "description": "Export job status, project metadata, and assets.",
```

Comment:
200 response is needed for SDK. Heath have added this in our earlier swaggers. There will be a validation error but that can be overriden.

Reply:
Reverted. How to override this specific validation error?
<img width="961" alt="image" src="https://user-images.githubusercontent.com/98181/193144481-72fc8e29-2c43-4980-b131-1f85b459eb4f.png">

Reply:
Fixed in example.